Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitFindController.cpp

```diff
@@ -60,9 +60,33 @@ class FindControllerTest: public WebViewTest {
         g_main_loop_run(m_mainLoop);
     }
 
+#if PLATFORM(GTK)
+    cairo_surface_t* takeSnapshotAndWaitUntilReady()
```

Comment:
```suggestion
    GRefPtr<cairo_surface_t> takeSnapshotAndWaitUntilReady()
```

Need to add a new GRefPtr specialization.

Reply:
No, GRefPtr is for glib types, cairo surface is not. We already have a RefPtr for cairo, but it's in WebCore, because WTF doesn't depend on cairo. The api tests only use the public API and WTF, but not WebCore API. That's the reason why for cairo types we manually ref/unref. Leaks in api tests that are no related to public API are not problematic, so I don't think it's worth the effort.